variables.tf

```diff
@@ -10,6 +10,11 @@ variable "deploy_hccm" {
   type        = bool
   default     = true
 }
+variable "deploy_csi_driver" {
+  description = "Deploy the csi-driver through Helm"
+  type        = bool
+  default     = false
```

Comment:
I've set this to false to keep the current behavior, but true would be fine as well :100:

Reply:
`false` works for me. HCCM is required in every cluster, either through the Helm chart or in the hccm repo by deploying the current code.

`csi-driver` is more optional and not required for core cluster functionality, so I prefer to have it disabled.

Reply:
Makes sense, thanks for the quick review/merge!